sdk/Dagger.SDK.SourceGenerator/Dagger.SDK.SourceGenerator/Code/CodeRenderer.cs

```diff
@@ -30,7 +30,7 @@ namespace Dagger.SDK;
 
                public class Scalar
                {
-                   public string Value;
+                   public string Value { get; set; }
```

Comment:
Was there reason this was field?

Reply:
In GraphQL, the `id` type such as, `EnvVariableID`, `ContainerID`, etc. represents as string value. And we need to render it to type. But problem I faced is I cannot find the best way to serialize string to type directly. So I ended up by introduce this property and write a json converter to serialize scalar string into this property.

Reply:
But it should stay as a C# `field` or `{ get; set; }`?

Reply:
Use property is fine for me. 👍